app/src/androidTest/java/org/kiwix/kiwixmobile/settings/KiwixSettingsFragmentTest.kt

```diff
@@ -33,17 +33,24 @@ import org.kiwix.kiwixmobile.testutils.TestUtils.TEST_PAUSE_MS
 import org.kiwix.kiwixmobile.utils.StandardActions
 
 class KiwixSettingsFragmentTest {
-  @Rule @JvmField var activityTestRule = ActivityTestRule(
+  @Rule
+  @JvmField
+  var activityTestRule = ActivityTestRule(
     KiwixMainActivity::class.java
   )
 
-  @Rule @JvmField var readPermissionRule: GrantPermissionRule =
+  @Rule
+  @JvmField
+  var readPermissionRule: GrantPermissionRule =
```

Comment:
@MohitMaliDeveloper Please commit the change code into the relevant commit. These code changes are unnecessary for this  PR and the commit as well.

Reply:
hi @gouri-panda ,
i did this for project consistency. okay i'm reverting this

Reply:
hi @gouri-panda , Done

Reply:
> i did this for project consistency. okay i'm reverting this

Yes!, if we are adding the codes. Changing the existing code without reason is overwhelming.